### PR TITLE
WFLY-5550 Distributable Web subsystem

### PR DESCRIPTION
**Pull request by pferraro**
https://issues.jboss.org/browse/WFLY-5550

### CODE REVIEW

**Comment by tommaso-borgato** [comment]
@pferraro I have one question: in this doc the routing option "RANKED" is mentioned but I cannot find it in [this|https://github.com/pferraro/wildfly/blob/web/docs/src/main/asciidoc/_high-availability/subsystem-support/Distributable_Web_Applications.adoc]; is it intentional?

**Comment by pferraro** [comment]
@tommaso-borgato I was originally going to include the RANKED routing option within the scope of this feature, but decided to move it into a separate body of work that is dependent on this one.  Thus, it should not be mentioned in this analysis document, as it will have its own.   I'll fix that.
The documentation link you pasted only reflects features that are already committed.  You should never see uncommitted features in committed documentation.

File: clustering/web/WFLY-5550_Distributable_Web_Subsystem.adoc [hunk 1]
```diff
@@ -0,0 +1,142 @@
+= <INSERT TITLE HERE>
```
Comment (jamezp) [inline clustering/web/WFLY-5550_Distributable_Web_Subsystem.adoc]: Could you add a title?

Comment (pferraro) [reply clustering/web/WFLY-5550_Distributable_Web_Subsystem.adoc]: So picky! :)

Comment (pferraro) [reply clustering/web/WFLY-5550_Distributable_Web_Subsystem.adoc]: Done.

**Comment by jamezp** [comment]
@tommaso-borgato Does this now seem reasonable to you? The feature freeze is 14.Nov.

**Pull request merged**